photon-lib/src/main/scala/com/linkedin/photon/ml/data/LabeledPoint.scala

```diff
@@ -33,42 +35,71 @@ class LabeledPoint(
   override val weight: Double = 1.0) extends DataPoint(features, weight) {
 
   /**
+   * A human readable string for this labeled point.
    *
-   * @return
+   * @return A string formatted for humans
    */
   override def toString: String = s"(label: $label offset: $offset weight: $weight features: $features)"
 
   /**
+   * Return a string for the label and feature values in this LabeledPoint, without adornments (for easy parsing and
+   * ingestion into other tools).
+   *
+   * The format is: label feature1Value feature2Value ...,
+   * all the data separated by a space.
+   *
+   * @return A machine-parsable, space separated string
+   */
+  def toRawString: String = s"$label ${features.toDenseVector.toArray.mkString(", ")}"
```

Comment:
Look into making this be the definition of `toString`, and adding the `Summarizable` trait and changing the current definition of `toString` to be the definition of `toSummaryString`. This would be in line with what other classes do, and avoid a third way of converting objects to strings in Photon-ML